src/Main.java

```diff
@@ -0,0 +1,24 @@
+public class Main extends Car  {
+
+         public static void main (String[] args) {
+             Car Audi = new Car();
+             Car BMW = new Car();
+             Car audi = new Car();
```

Comment:
variable names better named from small letter according with google or oracle code convention. And it is not good way to create a two instance with almost same names

Reply:
I executed what was given in the task:

Part 2

Create 3 cars objects
Audi, bmw, audi
set parameters to them and print country, model and maker